build_script.py

```diff
@@ -81,11 +81,17 @@ def main():
                         action="store_true",
                         dest="test",
                         default=False)
+    parser.add_argument("--arch",
+                        help="target architecture",
+                        action="store",
+                        dest="arch",
+                        default=None)
```

Comment:
It looks like the build invocation below uses `arch`. Shouldn't `--arch` be marked as `required=True` here?